src/extensions/advancedcharts/chart/choropleth/ChoroplethMapChart.tsx

```diff
@@ -20,7 +20,7 @@ const NeoChoroplethMapChart = (props: ChartProps) => {
     // TODO Think of a way to make it configurable to fetch vector data.
     // It makes sense to ship this JSON with NeoDash deployments that are behind some firewall and can't access this site.
     useEffect(() => {
-        fetch("https://raw.githubusercontent.com/plouc/nivo/master/website/src/data/components/geo/world_countries.json")
+        fetch("https://raw.githubusercontent.com/BennuFire/neodashChoroplethMaps/main/world_DE")
```

Comment:
Is this set to this URL on purpose ?

Reply:
I've created a seperate repository to store large files outside of the core repo.
Changed the URL to here:
https://raw.githubusercontent.com/neo4j-labs/neodash-static/main/world_polymap.json
Lets see later if we want to revise this strategy.